### PR TITLE
Neo_view template population

### PR DESCRIPTION
**Pull request by ppapadeas**

### CODE REVIEW

File: anapneo/urls.py [hunk 1]
```diff
@@ -12,6 +12,12 @@
     url(r'^admin/', include(admin.site.urls)),
 )
 
+media_url = settings.MEDIA_URL.lstrip('/').rstrip('/')
+urlpatterns += patterns('',
+    url(r'^%s/(?P<path>.*)$' % media_url, 'django.views.static.serve',
+        {'document_root': settings.MEDIA_ROOT}),
+)
+
```
Comment (comzeradd) [inline anapneo/urls.py:15]: This is needed only on local development, so please include it on "if settings.DEBUG:" clause along with static files.

Comment (ppapadeas) [inline anapneo/urls.py:15]: This is also needed in production for upload, no?

Comment (comzeradd) [inline anapneo/urls.py:15]: Nope. These should be served with whatever way we serve statics too. This is needed only by Django's development server.

File: anapneo/static/js/neo_view.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+function check_submit(v) {
+    "use strict";
+    alert('dsa');
+    $('input').removeAttr('checked');
+    $('input[value=' + v + ']').prop('checked', true);
+    $('#voting_form').submit();
+}
+
+$(document).ready(function () {
+	if ($('input[value="-1"]').is(':checked')) {
```
Comment (comzeradd) [inline anapneo/static/js/neo_view.js]: Since you are going to amend this commit, use 4 spaces here :P

**Comment by comzeradd** [comment]
r+
:cocktail:

**Pull request merged**